File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    // Initialize height variable
    int height;

    // Prompt user for height
    do
    {
        height = get_int("Height: ");
    }
    // Keep prompting user if height value doesn't meet spec
    while(height < 0 || height > 23);

        for(int i = 1; i <= height; i++)
        {
            // Print spaces
            for(int j = 0; j < (height - i); j++)
            {
                printf(" ");
            }

            // Print left blocks
            for(int k = 0; k < i; k++)
            {
                printf("#");
            }

            // Print gap
            printf("  ");

            // Print right blocks
            for(int k = 0; k < i; k++)
            {
                printf("#");
            }

            // Print new lines
            printf("\n");
        }

        return 0;
}

File: pset6/mario/mario.py
from cs50 import get_int

height = get_int('Height: ')


while height < 0 or height > 23:
    print('Enter an integer between 0 and 23')
    height = get_int('Height: ')

for i in range(height):
    # print(f'i:{i}')
    # print(f'height: {height}')

    # Print spaces
    for j in range(1, (height - i)):
        print(' ', end='')
        # print(f'j:{j}')

    # Print blocks
    for k in range(i+1):
        print('#', end='')
        # print(f'k:{k}')

    print('  ', end='')

    # # Print blocks
    for k in range(i+1):
        print('#', end='')

    print()


